src/main/java/org/tikv/common/region/AbstractRegionStoreClient.java

```diff
@@ -152,12 +154,25 @@ public boolean onStoreUnreachable(BackOffer backOffer) {
     return false;
   }
 
-  protected Kvrpcpb.Context makeContext(TiStoreType storeType) {
-    return region.getReplicaContext(java.util.Collections.emptySet(), storeType);
+  private Kvrpcpb.Context addTraceId(Kvrpcpb.Context context, SlowLog slowLog) {
+    long traceId = slowLog.getTraceId();
+    return Kvrpcpb.Context.newBuilder(context)
+        .setTraceContext(
+            Tracepb.TraceContext.newBuilder()
+                .setDurationThresholdMs((int) slowLog.getThresholdMS())
```

Comment:
I think we should set the threshold for tikv less than the client. Maybe half of the client threshold? @marsishandsome what do you think?

Reply:
Should we make the factor(client threshold / tikv threshold) configurable?

Reply:
Maybe we can leave it later

Reply:
Sounds reasonable